src/docs/product/data-management-settings/dynamic-sampling/index.mdx

```diff
@@ -43,6 +44,10 @@ While prioritizing data from the latest release, we also take into consideration
 
 We prioritize data from development environments because these environments typically generate a relatively small number of transactions compared to your production environments. As a result, to gain meaningful insights during your testing phase, we prioritize transaction data from these environments.
 
+### Key Transactions
+
+For most projects there are only a handful of transactions that you really care about. We prioritize key, or starred, transactions over others, so you can have more visibility into your important data, rather than sample all transactions in equivalent manner.
```

Comment:
This is much clearer. The sentence gets too long with that last clause and it's kind of implied anyway by the fact that we're saying we prioritize them, so I removed it. Also, I added the word "typically" so that it leaves room for the fact that this might not be the case for some folks.

```suggestion
Typically, for most projects there are only a few transactions that you really care about. We prioritize key, or starred, transactions over others, so you can have more visibility into your important data.
```